apps/user/templates/user/profile.html

```diff
@@ -30,7 +30,7 @@ <h2>Progress report on all your subjects.</h2>
       {% for attempt in page_obj %}
       <tr>
         <td>{{ attempt.created }}</td>
-        <td>{{ attempt.exam.name }}</td>
+        <td>{{attempt.exam.subject.name}} {{ attempt.exam.name }}</td>
```

Comment:
May be worth adding `select_related` to the get_queryset() https://github.com/kasya/sim2/blob/main/apps/user/views.py#L71